src/main/resources/config/liquibase/changelog/20230913162100_dashboard_aud.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <!--
+        Added revinfo table for hibernate-envers.
+    -->
+    <changeSet id="20230913162100-1" author="dbahazhkov">
+
+        <createTable tableName="revinfo">
+            <column name="rev" type="bigint">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="revtstmp" type="TIMESTAMP(6) WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_modified_by" type="varchar(256)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+    </changeSet>
+
+    <changeSet id="20230913162100-2" author="dbahazhkov">
+        <addColumn tableName="dashboard">
+            <column name="created_by" type="varchar(256)"/>
```

Comment:
why do you add these columns to Dashboard table as you already have them in AUD table?

Reply:
@amedvedchuk i thought that it would be easier to get information about the latest changes right away, without an unnecessary query to another table

Reply:
@d-bagazhkov could you please clarify where this data filled? I didn't see this fields at the domain object: com.icthh.xm.ms.dashboard.domain.Dashboard